components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -813,12 +813,10 @@ func prepareIDELaunch(cfg *Config, ideConfig *IDEConfig) *exec.Cmd {
 
 	// Add default args for IDE (not desktop IDE) to be backwards compatible
 	if ideConfig.Entrypoint == "/ide/startup.sh" && len(args) == 0 {
-		args = append(args, "{WORKSPACEROOT}")
```

Comment:
Safe to remove as workspace to open in calculated in `gitpod/workbench.ts`

Reply:
https://github.com/gitpod-io/openvscode-server/blob/gp-code/main/src/vs/gitpod/browser/workbench/workbench.ts

Reply:
Sure it's on the [openvscode-server repo](https://github.com/gitpod-io/openvscode-server/blob/40c9a1fb052740734bf465b98032f1a50404c042/src/vs/gitpod/browser/workbench/workbench.ts#L533-L546)